Cross Region VM Restore Points/Cross Region Creation of Restore Points/Cross Region Creation of VM Restore Points.md

```diff
@@ -0,0 +1,456 @@
+            "linuxConfiguration": {
+                "disablePasswordAuthentication": "true|false",
+                "ssh": {
+                    "publicKeys": [
+                        {
+                            "path": "Path-Where-To-Place-Public-Key-On-VM",
+                            "keyData": "PEM-Encoded-public-key-file"
+                        }
+                    ]
+                }
+            }
+        },
+        "storageProfile": {
+            "osDisk": {
+                "osType": "Windows|Linux",
+                "name": "OSDiskName",
+                "diskSizeGB": "10",
+                "caching": "ReadWrite",
+                "managedDisk": {
+                    "id": "CSM Id of the managed disk",
+                    "storageAccountType": "Standard_LRS"
+                },
+                "diskRestorePoint": {
+                    "id": "/subscriptions/<subId>/resourceGroups/<rgName>/restorePointCollections/<rpcName>/restorePoints/<rpName>/diskRestorePoints/<diskRestorePointName>"
+                }
+            },
+            "dataDisks": [
+                {
+                    "lun": "0",
+                    "name": "datadisk0",
+                    "diskSizeGB": "10",
+                    "caching": "ReadWrite",
+                    "managedDisk": {
+                        "id": "CSM Id of the managed disk",
+                        "storageAccountType": "Standard_LRS"
+                    },
+                    "diskRestorePoint": {
+                        "id": "/subscriptions/<subId>/resourceGroups/<rgName>/restorePointCollections/<rpcName>/restorePoints/<rpName>/diskRestorePoints/<diskRestorePointName>"
+                    }
+                }
+            ]
+        },
+        "diagnosticsProfile": {
+            "bootDiagnostics": {
+                "enabled": true,
+                "storageUri": " http://storageaccount.blob.core.windows.net/"
+            }
+        }
+    },
+    "optionalProperties": "opaque bag of properties to be passed to extension",
+    "consistencyMode": "CrashConsistent | FileSystemConsistent | ApplicationConsistent",
+    "provisioningState": "Succeeded | Failed | Creating | Deleting",
+    "provisioningDetails": {
+        "creationTime": "Creation Time of Restore point in UTC",
+        "totalUsedSizeInBytes": "25",
+        "statusCode": "status code reported by the extension",
+        "statusMessage": "status message reported by the extension"
+    }
+}
+```
+
+
+### Get VM Restore Points With Copy/Replication Status
+Once creation of VM Restore Points is initiated, you can track the data copy status by calling GET instance View (?$expand=instanceView) on the VM Restore Point.
+
+#### URI Request
+```
+GET https://management.azure.com/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/restorePointCollections/{restorePointCollectionName}/restorePoints/{restorePointName}?$expand=instanceView?api-version={api-version}
```

Comment:
?$expand=instanceView**?**api-version={api-version} is not correct... it should be "?$expand=instanceView**&**api-version={api-version}"

Reply:
Done